src/AsmResolver.Workspaces.DotNet/DotNetRelations.cs

```diff
@@ -47,13 +47,28 @@ public static class DotNetRelations
             "ReferenceMember",
             new Guid("ce11d2f6-a423-429d-ad37-2f073fdf63be"));
 
+        /// <summary>
+        /// Describes the relationship between a method definition and its specification.
+        /// </summary>
+        public static readonly ObjectRelation<MethodDefinition, MethodSpecification> ReferenceMethodSpecification = new(
```

Comment:
Shouldn't this be a `IMethodDefOrRef` to `MethodSpecification`,  instead of `MethodDefinition` to `MethodSpecification`? A `MethodSpecification`  can also instantiate member references.